*** fish-shell/Makefile.in	2013-04-11 08:51:53.404966847 +0200
--- aidsfish/Makefile.in	2013-04-11 08:51:11.082842866 +0200
***************
*** 510,532 ****
  .PHONY: check-uninstall
  
  
- # seq used to be a shell script that we would install
- # Now we just look for previously installed seqs, and erase them
- # No big deal if it fails
- # same with set_color, which is now a builtin.
- # Invoke set_color with -v to make sure it's ours.
- cleanup_old_binaries:
- 	SEQLOC=`which seq`;\
- 	if test -x "$$SEQLOC" && grep -q '\(^#!/.*/fish\|^#!/usr/bin/env fish\)' "$$SEQLOC"; then\
- 		rm -f "$$SEQLOC";\
- 	fi;\
- 	SETCOLOR_LOC=`which set_color`;\
- 	if test -x "$$SETCOLOR_LOC" && $$SETCOLOR_LOC -v 2>&1 >/dev/null | grep -q "^set_color, version "; then\
- 		rm -f "$$SETCOLOR_LOC";\
- 	fi;\
- 	true;
- .PHONY: cleanup_old_binaries
- 
  #
  # This check makes sure that the install-sh script is executable. The
  # darcs repo doesn't preserve the executable bit, so this needs to be
--- 510,515 ----
***************
*** 542,548 ****
  # Try to install after checking for incompatible installed versions.
  #
  
! install: all cleanup_old_binaries install-sh check-uninstall install-force
  .PHONY: install
  
  #
--- 525,531 ----
  # Try to install after checking for incompatible installed versions.
  #
  
! install: all install-sh check-uninstall install-force
  .PHONY: install
  
  #
